src/controllers/action-queue/redux/actions.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import { type Dispatch, type GetState } from '../../../types/reduxTypes'
+import { type ActionQueueStore, makeActionQueueStore } from '../ActionQueueStore'
+import { type ActionProgram, type ActionProgramState, type ActionQueueItem, type ActionQueueMap } from '../types'
+
+type LoadActionQueueStateAction = {
+  type: 'ACTION_QUEUE/LOAD_QUEUE',
+  data: ActionQueueMap
+}
+
+export const loadActionQueueState = (data: ActionQueueMap): LoadActionQueueStateAction => {
+  return {
+    type: 'ACTION_QUEUE/LOAD_QUEUE',
+    data
+  }
+}
```

Comment:
We are also trying to get rid of action creator methods like this. Just `dispatch({ ...whatever })` directly whenever possible - it shows that there are no secret side effects, which makes the program easier to reason about.

Reply:
Hmm, I know what you mean, but you may want to explain this one to me so I "get what you mean".

Reply:
Got it after the explanation. 👍